### PR TITLE
Task/Accomodation service test

### PR DESCRIPTION
**Pull request by VolandevlodD**

### CODE REVIEW

File: src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        AccommodationDto result = accommodationService.create(createAccommodationRequestDto);
+
+        assertNotNull(result);
+        assertEquals(accommodationDto, result);
+        verify(accommodationRepository, times(1)).save(any(Accommodation.class));
+    }
+
+    @Test
+    @DisplayName("Get all accommodations should return a list of accommodations")
+    void getAllAccommodationsShouldReturnListOfAccommodations() {
```
Comment (TonyH277) [inline src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java:106]:
```suggestion
    void getAllAccommodations_ShouldReturnListOfAccommodations() {
```

**Review by TonyH277 — CHANGES_REQUESTED** [review]
Nice! Pls remove default value of "times(1)" and change test naming to "testMethodName_InputData_ExpectedBehavior"

File: src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+        assertNotNull(result);
+        assertEquals(accommodationDto, result);
+        verify(accommodationRepository)
+                .findByIdWithAddressAndAmenities(accommodation.getId());
+    }
+
+    @Test
+    @DisplayName("Update accommodation should save and return the updated accommodation")
+    void updateAccommodation_ShouldSaveAndReturnUpdatedAccommodation() {
```
Comment (TonyH277) [inline src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java:140]:
```suggestion
    void updateAccommodation_ReturnsUpdatedAccommodation() {
```

File: src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        );
+
+        assertNotNull(result);
+        assertEquals(accommodationDto, result);
+        verify(accommodationRepository).save(any(Accommodation.class));
+    }
+
+    @Test
+    @DisplayName("Patch accommodation should save and return the updated accommodation")
+    void patchAccommodation_ShouldSaveAndReturnUpdatedAccommodation() {
```
Comment (TonyH277) [inline src/test/java/com/ua/accommodation/service/impl/AccommodationServiceImplTest.java:161]:
```suggestion
    void patchAccommodation_ReturnsUpdatedAccommodation() {
```

**Review by TonyH277 — APPROVED** [review]
Nice!

**Review by MolchanovAlexander — APPROVED** [review]
Good job!

**Pull request merged**